src/routes/auth.routes.js

```diff
@@ -0,0 +1,31 @@
+import express from 'express';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import jwt from 'jsonwebtoken';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import bcrypt from 'bcrypt';
+import db from '../database/models';
+
+const router = express.Router();
+
+// eslint-disable-next-line operator-linebreak
+const JWT_SECRET =
+  'goK!pusp6ThEdURUtRenOwUhAsWUCLheBazl!uJLPlS8EbreWLdrupIwabRAsiBu';
```

Comment:
Please get the token from `.env` you should not push sensitive information like that